brave/brave-core
# Introduce "Expand vertical tabs per window"

When this preference is on, clicking toggle button will affect only the browser window that the toggle button is in.


Resolves https://github.com/brave/brave-browser/issues/36224



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Go to settings > Appearance > tabs
* Turn on "Expand vertical tabs per window" and vertical tab mode
* Open multiple windows
* Check if clicking toggle button affects only the browser that toggle button is in
